src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24489_Shell.cs

```diff
@@ -0,0 +1,94 @@
+#if MACCATALYST || WINDOWS
+
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24489_Shell : _IssuesUITest
+	{
+		public Issue24489_Shell(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "TitleBar with NavigationBar - Shell";
+
+		protected override bool ResetAfterEachTest => true;
+
+		[Test]
+		[Category(UITestCategories.Window)]
+		public void TitleBarWithSmallHeightShell()
```

Comment:
Tested manually and looks good to me, but these snapshots are pending. Could you commit the images?